apps/learning-snippets/src/test/tree-widget/SubTreePaths.test.tsx

```diff
@@ -0,0 +1,116 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+/* eslint-disable import/no-duplicates */
+/* eslint-disable @typescript-eslint/no-shadow */
+
+import { expect } from "chai";
+import { useCallback } from "react";
+import sinon from "sinon";
+import { UiFramework } from "@itwin/appui-react";
+import { IModelApp } from "@itwin/core-frontend";
+import { useModelsTree, VisibilityTree, VisibilityTreeRenderer } from "@itwin/tree-widget-react";
+import { createStorage } from "@itwin/unified-selection";
+import { cleanup, render, waitFor } from "@testing-library/react";
+import { buildIModel, insertPhysicalElement, insertPhysicalModelWithPartition, insertSpatialCategory } from "../../utils/IModelUtils.js";
+import { initializeLearningSnippetsTests, terminateLearningSnippetsTests } from "../../utils/InitializationUtils.js";
+import { getSchemaContext, getTestViewer, mockGetBoundingClientRect, TreeWidgetTestUtils } from "../../utils/TreeWidgetTestUtils.js";
+
+import type { SelectionStorage } from "@itwin/unified-selection";
+import type { IModelConnection, Viewport } from "@itwin/core-frontend";
+import type { InstanceKey } from "@itwin/presentation-common";
+
+// __PUBLISH_EXTRACT_START__ TreeWidget.GetSubTreePathsComponentWithTargetItemsExample
+type UseModelsTreeProps = Parameters<typeof useModelsTree>[0];
```

Comment:
`@itwin/presentation-shared` has a `Props` utility that makes this more readable:

```suggestion
type UseModelsTreeProps = Props<typeof useModelsTree>;
```